app/src/main/kotlin/com/wire/android/ui/theme/WireColorPalette.kt

```diff
@@ -1,61 +1,12 @@
-    val error: Color,                           val onError: Color,
-    val errorOutline: Color, //TODO missing on designs
-    val warning: Color,                         val onWarning: Color,
-    val positive: Color,                        val onPositive: Color,
-    val background: Color,                      val onBackground: Color,
-    val backgroundVariant: Color,               val onBackgroundVariant: Color,
-    val surface: Color,                         val onSurface: Color,
-    val primaryButtonEnabled: Color,            val onPrimaryButtonEnabled: Color,
-    val primaryButtonDisabled: Color,           val onPrimaryButtonDisabled: Color,
-    val primaryButtonSelected: Color,           val onPrimaryButtonSelected: Color,
-    val primaryButtonRipple: Color,
-    val secondaryButtonEnabled: Color,          val onSecondaryButtonEnabled: Color,
-    val secondaryButtonEnabledOutline: Color,
-    val secondaryButtonDisabled: Color,         val onSecondaryButtonDisabled: Color,
-    val secondaryButtonDisabledOutline: Color,
-    val secondaryButtonSelected: Color,         val onSecondaryButtonSelected: Color,
-    val secondaryButtonSelectedOutline: Color,
-    val secondaryButtonRipple: Color,
-    val tertiaryButtonEnabled: Color,           val onTertiaryButtonEnabled: Color,
-    val tertiaryButtonDisabled: Color,          val onTertiaryButtonDisabled: Color,
-    val tertiaryButtonSelected: Color,          val onTertiaryButtonSelected: Color,
-    val tertiaryButtonSelectedOutline: Color,
-    val tertiaryButtonRipple: Color,
-    val divider: Color,
-    val secondaryText: Color,
-    val labelText: Color, //TODO missing on designs
-    val badge: Color,                           val onBadge: Color //TODO missing on designs
-) {
-    fun toColorScheme(): ColorScheme = ColorScheme(
-        primary = primary,                              onPrimary = onPrimary,
-        primaryContainer = secondaryButtonSelected,     onPrimaryContainer = onSecondaryButtonSelected,
-        inversePrimary = secondaryButtonSelected,
-        secondary = badge,                              onSecondary = onBadge,
-        secondaryContainer = backgroundVariant,         onSecondaryContainer = onBackgroundVariant,
-        tertiary = primary,                             onTertiary = onPrimary,
-        tertiaryContainer = secondaryButtonSelected,    onTertiaryContainer = onSecondaryButtonSelected,
-        background = background,                        onBackground = onBackground,
-        surface = surface,                              onSurface = onSurface,
-        surfaceVariant = backgroundVariant,             onSurfaceVariant = onBackgroundVariant,
-        inverseSurface = onPrimaryButtonDisabled,       inverseOnSurface = Color.White,
-        error = error,                                  onError = onError,
-        errorContainer = errorOutline,                  onErrorContainer = error,
-        outline = divider
-    )
-}
-
+@PackagePrivate
```

Comment:
Wouldn't `internal` be sufficient as a visibility modifier here?

Reply:
I just wanted to make sure that no-one will use such class because we should be using theme-dependent colors (`MaterialTheme.wireColors.color`) instead of referring directly to these color palette and `internal` doesn't ensure that